components/authn-service/Makefile

```diff
@@ -63,4 +63,11 @@ kill_docker_pg:
 proto:
 	cd ../../ && hab studio run 'source .studiorc; compile_go_protobuf_component authn-service'
 
+spellcheck:
+	@pushd ../.. > /dev/null; \
+	./scripts/spellcheck.sh --wordsOnly components/authn-service; \
+	EXIT_CODE=$$?; \
+	popd > /dev/null; \
+	exit $$EXIT_CODE
```

Comment:
I wonder if there is a way to put this in Makefile.common_go so we don't have to copy it around as much?

Reply:
I agree, I think I'd rather see this added to every project in a standard way rather than through copy-pasting.

Reply:
Figured out the variable machinations to do that and just pushed a commit.

Reply:
@phiggins I used `make spell` for the common functionality. But note that now `automate-chef-io` has its own private `make spellcheck` plus this common one `make spell`. What do you think about renaming the private one to be `make spell-plus-protos` so it will be much clearer what the private one is for?

Reply:
I'd be ok renaming, but I think I'd prefer something revealing the why more than the what, eg `make spell-all` or something.

Would it be possible to allow each service to customize what was checked like how the go linter does? Definitely not something you need to figure out today. :)